pipeline_steps/common.groovy

```diff
@@ -2107,6 +2112,40 @@ void createPrWhisperer(String name, String repoUrl, String projectsFile){
   } // if
 }
 
+/**
+* Create a Checkmarx code scan job when a new Repo is added.
```

Comment:
What about existing projects?

Reply:
They will be covered separately, @chundu is keen that teams create PRs to add scans for each repo so they can define which are in scope for PCI.